app/controllers/registrations_controller.rb

```diff
@@ -22,7 +22,8 @@ def account_update_params
       :agency_id,
       :password,
       :password_confirmation,
-      :current_password
+      :current_password,
+      :approved
```

Comment:
Ah.  Yes great catch, missed that one... will fix.    BTW:  I'm gonna need to change this cause I was using turbolinks and that apparently got ripped out for some reason.